corehq/apps/auditcare/tests/test_export.py

```diff
@@ -250,6 +252,51 @@ def test_related_query_error(self):
     def test_get_sql_start_date(self):
         self.assertEqual(get_sql_start_date(), datetime(2021, 2, 1, 3))
 
+    def test_get_domain_first_access_times(self):
+
+        def create_session_events(domain):
+            login_event = dict(
+                user=self.username,
+                domain=domain,
```

Comment:
I think this is always overwritten, so this line can be removed.

Reply:
It is always overwritten, but it needs to be in `login_event` (the function returns this).  Perhaps the code would be clearer if I omit it while instantiating the dict and then add it just before the `return` statement?  I could be convinced either way.

Reply:
> it needs to be in `login_event`

Ah, I missed that. Fine to leave as is.